src/Slim/Middleware/Session.php

```diff
@@ -85,6 +85,8 @@ protected function startSession()
     {
         $settings = $this->settings;
         $name = $settings['name'];
+        
+        session_write_close();
```

Comment:
Nope. It will recreate a session each time.
Please see the comment I left at #32 which says what's needed.